index.js

```diff
@@ -0,0 +1,46 @@
+'use strict'
+
+const Base = require('bfx-facs-base')
+
+class GrcSlack extends Base {
+  constructor (caller, opts, ctx) {
+    super(caller, opts, ctx)
+
+    this.name = 'grc-slack'
+    this._hasConf = true
+    this.init()
+
+    if (opts.conf) this.conf = opts.conf
+  }
+
+  message (message, reqChannel) {
+    if (!this.conf.enable) return false
+    const slack = this.conf
+    const worker = slack.worker || 'rest:ext:slack'
+    const maxLength = slack.max_length || 1024
+    const env = (slack.env) ? `Env: ${slack.env}, ` : ''
+    const rawText = env + message
+    const text = (maxLength) ? rawText.substr(0, maxLength) : rawText
+    const channel = reqChannel || slack.channel
+    const send = [{ channel, text }]
+
+    return this.caller.grc_bfx.req(
+      worker,
+      'postSlackMsg',
+      send,
+      { timeout: 10000 })
+  }
+
+  logError (err, notes, reqChannel) {
```

Comment:
maybe we can improve it a bit to be more generic by passing params. e.g. in celsius I have config like this:
```js
  /**
   * @param {Grc} grc
   * @param {string} channel
   * @param  {...any} params
   */
  static slacklog (grc, channel, ...params) {
    const originalText = formatSec(...params)
    const slackText = originalText.substr(0, 1024)
    Logger.error(originalText)

    grc.req('rest:ext:slack', 'postSlackMsg', [{ channel, text: slackText }], {}, (err, res) => {
      if (err || !(res || {}).ok) {
        Logger.error('slacklog failed: %s', err ? err.toString() : 'RES_NOT_OK')
      }
    })
  }
```

so we could to it something like this:
```js
logError (reqChannel, ...params) {
    const text = util.format(params)
    return this.message(text, reqChannel)
}
```

also currently if (err instanceof Error) return e.stack removes e.message property which hides the issue info and util.format would keep it appropriate as it is in case of console.log(err)